src/libraries/System.Net.Mail/src/System/Net/Mime/MediaTypeMap.cs

```diff
@@ -0,0 +1,863 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+
+namespace System.Net.Mime
+{
+    /// <summary>
+    /// Provides methods for mapping between file extensions and media types (MIME types).
+    /// </summary>
+    public static class MediaTypeMap
+    {
+        private const int ExtensionToMediaTypeCount = 429;
+        private const int MediaTypeToExtensionCount = 278;
```

Comment:
```suggestion
        private const int ExtensionToMediaTypeCount = 430;
        private const int MediaTypeToExtensionCount = 279;
```
To account for the cbor change